src/controllers/historycontroller.js

```diff
@@ -13,34 +13,26 @@ class HistoryController extends Controller
     {
         if (entry instanceof Customer)
         {
-            this.addCustomer(entry);
+            this._loadTemplate(
+                "src/views/template/history/customer.html",
+                "#history-events",
+                entry,
+                false,
+                true
+            );
         }
 
         if (entry instanceof Container)
         {
-            this.addContainer(entry);
+            this._loadTemplate(
```

Comment:
No, don't add the call to `_loadTemplate` twice (i.e. in different `if`s). Set the loading string based on the `instanceof` check, and then have this piece of code in your code only once with that string.